server/services/config/conditions/v2/conditions.ts

```diff
@@ -21,8 +20,27 @@ enum GroupName {
   POST_SENTENCE_SUPERVISION_ONLY = 'Post-sentence supervision only',
 }
 
-export const getPersistedAbuseAndBehaviours = (conditions: AdditionalConditions) =>
-  (conditions as AdditionalConditionsV2)?.COMPLY_REQUIREMENTS?.abuseAndBehaviours
+export const modifyAdditionalConditions = (someConditions: AdditionalConditions) => {
+  const conditions = someConditions as AdditionalConditionsV2
+
+  const abuseAndBehavioursConditions = conditions?.COMPLY_REQUIREMENTS?.abuseAndBehaviours
+
+  if (Array.isArray(abuseAndBehavioursConditions)) {
+    Object.assign(
+      abuseAndBehavioursConditions,
+      abuseAndBehavioursConditions.map((condition, index) => (index > 0 ? ` ${condition}` : condition))
+    )
+  }
+
+  const appointmentProfessions = conditions?.ATTEND_ALL?.appointmentProfessions
+
+  if (Array.isArray(appointmentProfessions)) {
+    Object.assign(
```

Comment:
TBH - I don't really understand this, copied the existing behaviour - I've used it for objects but not arrays. seems to override the assignment of the existing variable with a new valu